tools/generate_json_rpc_docs.py

```diff
@@ -181,16 +181,16 @@ def to_markdown(self):
 items.sort(key=lambda item: item.name)
 
 PREAMBLE = """
-# Jamulus JSON-RPC Documentation
+# Jamulus JSON-RPC Server Documentation
 
 <!--
 This file is automatically generated from the source code.
 Do not edit this file manually.
 See `tools/generate_json_rpc_docs.py` for details.
 -->
 
-A JSON-RPC interface is available for both Jamulus client and server to allow programmatic access.
-To use the JSON-RPC interface, run Jamulus with the `--jsonrpcport <port> --jsonrpcsecretfile /file/with/a/secret.txt` options.
+A JSON-RPC server can be added to both Jamulus client and Jamulus server to allow programmatic access.
```

Comment:
At _runtime_, it's nothing is being "adding" - the service is there.  The `--jsonrpc...` options _enable_ the service.

Reply:
Also, a style comment: "Jamulus client" and "Jamulus server" should be "Jamulus Client" and "Jamulus Server" to comply with our terminology guide.  (However, "JSON-RPC server" would use title case - only capitalising the "server" in titles.  I've a marginal preference for "service", as "server" suggests it's separate.)

I guess this guide hasn't been through style guide compliance check yet, though.

Reply:
I defer to the style of the document I'm in. 

Does the JSON-RPC server reveal itself in any way when no port-password params are present? Add those, get it?

My bias against **enable** started about 20 years ago. Now I frequently prefer **adds**, **lets**, or **turns on**.  I often ask myself, "What would a 4-year-old understand?" Alright, let's examine your claim: The service _is there_, yet it can be _enabled_? Is it there and disabled? If userspace can't see its enabled-disabled state, then I don't find its there-ness noteworthy. Google's style guide relegates _enabled_ to UI states, where a disabled control has a visual indication in userspace. Is there any such state indication here? What can I do with a disabled JSON-RPC server? Can I enable it in some way other than providing these parameters? Are you concerned that a systems-level developer would be misled by this oversimplification presented to value-add developers?

Reply:
> Is there any such state indication here?

Assuming the build ran with the JSONRPC code, then there _should_ be a control in client and server UI allow control over use.  That there isn't is part of the JSONRPC implementation not being completed, rather than design.  Basing design decisions on an incomplete implementation isn't good practice...

Reply:
Technical writing is not design decisions. It's communication decisions. Public-facing content should reflect the product shipping, in the most easily understood terms. I like to update it when new features are finally live, as a last and savory step. In general, futureware should have nearly no impact on live documentation. At least that's a rule of thumb I try to use. We don't have to worry about confusing our readers when we switch later to coverage of a more complex featureset. Our time with readers is incredibly brief, and we often have just one chance to be understood adequately. What may seem like oversimplification is often a service to readers. (I also simplify to an extreme degree when coding UI, because I think the same cognition principles apply there. I'm a little obsessed with how closely I can say the bare minimum or simplest thing without impeding outcomes.) Another term I've heard for straddling this racy oversimplification line is _marketecture_.

Reply:
Fair point. Did you already read our style and tone guide? https://jamulus.io/contribute/Style-and-Tone

I think that's the place we should remember to stay close to. Probably worth discussing that in a GH discussion or issue.